### PR TITLE
KK-266 | Update README.md

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

**Comment by t0mim** [comment]
Looks good 👍

File: README.md [hunk 1]
```diff
@@ -82,3 +95,15 @@ Or you can use [`pre-commit`](https://pre-commit.com/) to quickly format your co
...
 
 After that, formatting hooks will run against all changed files before committing
+
+## Jira
+
+- Link to Backlog:
+https://helsinkisolutionoffice.atlassian.net/secure/RapidBoard.jspa?rapidView=28&projectKey=KK&view=planning.nodetail&epics=visible&issueLimit=100
+
+- Active Sprint:
+https://helsinkisolutionoffice.atlassian.net/secure/RapidBoard.jspa?rapidView=28&projectKey=KK
```
Comment (tuomas777) [inline README.md]: Do we actually want to add links to non-public things here? I don't really like it :)

Comment (quyenlq) [reply README.md]: Hmn I thought these are public and can be accessed without permission but these aren't. Then I don't think it's necessary here, also the confluence page at the beginning should be removed as well

**Comment by badrange** [comment]
@quyenlq Please change to using "Culture Kids" instead of Godchildren, we have a new and better English name for the project

**Pull request merged**